### PR TITLE
deploy market wordpress on ecs instance

### PR DESCRIPTION
**Pull request by lixue323**

### CODE REVIEW

File: README-CN.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+  ecs_instance_name          = "wordpress-instance"
+  ecs_instance_password      = "YourPassword123"
+  ecs_instance_type          = "ecs.sn1ne.large"
+  system_disk_category       = "cloud_efficiency"
+  security_group_ids         = ["sg-45678"]
+  vswitch_id                 = "vsw-345678"
+  internet_max_bandwidth_out = 50
+  image_id = "m-2ze69pmuqqu88qn5h47u"
+}
```
Comment (zql-tqy) [inline README-CN.md:25]: “m-2ze69pmuqqu88qn5h47u”改成“m-2ze69pmuqqu88qn5****”，然后把格式调整一下

File: README-CN.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+terraform-alicloud-market-wordpress
+=====================================================================
+
+本 Terraform 模块将使用 Wordpress 镜像来创建 ECS 实例，此实例用来启动 Wordpress。
+
+## Terraform 版本
+
+本模板要求使用版本 Terraform 0.13 和 阿里云 Provider 1.57.0+。
+
```
Comment (zql-tqy) [inline README-CN.md:9]: 确定一下这个版本

File: README.md [hunk 3]
```diff
@@ -1,2 +1,52 @@
...
+## Terraform versions
+
+This module requires Terraform 0.13 and Terraform Provider AliCloud 1.57.0+.
+
+## Usage
+
+```hcl
+module "market-wordpress" {
+  source = "terraform-alicloud-modules/market-wordpress/alicloud"
+  region = var.region
```
Comment (Rich-yi) [inline README.md]: readme里面示例参数写出来，不要用变量

File: README.md [hunk 4]
```diff
@@ -1,2 +1,52 @@
...
+
+  ecs_instance_name          = "wordpress-instance"
+  ecs_instance_password      = "YourPassword123"
+  ecs_instance_type          = "ecs.sn1ne.large"
+  system_disk_category       = "cloud_efficiency"
+  security_group_ids         = ["sg-45678"]
+  vswitch_id                 = "vsw-345678"
+  internet_max_bandwidth_out = 50
+  image_id = "m-2ze69pmuqqu88qn5h47u"
```
Comment (Rich-yi) [inline README.md]: image_id = "m-2ze69pmuqqu88qn5h47u" ===> image_id = "m-2ze69pmuxxxxxxxxxx"

File: main.tf [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+  current = true
+}
+
+module "ecs-instance" {
+  source                  = "alibaba/ecs-instance/alicloud"
+  profile                 = var.profile != "" ? var.profile : null
+  shared_credentials_file = var.shared_credentials_file != "" ? var.shared_credentials_file : null
+  region                  = var.region != "" ? var.region : null
+  skip_region_validation  = var.skip_region_validation
+
```
Comment (Rich-yi) [inline main.tf:17]: profile                 = var.profile != "" ? var.profile : null
shared_credentials_file = var.shared_credentials_file != "" ? var.shared_credentials_file : null
skip_region_validation  = var.skip_region_validation
这几个不用写进来吧，region传一个变量就行了

File: variables.tf [hunk 6]
```diff
@@ -0,0 +1,101 @@
+variable "region" {
```
Comment (Rich-yi) [inline variables.tf:1]: variables全部把type加上

File: main.tf [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+  region                  = var.region != "" ? var.region : null
+  skip_region_validation  = var.skip_region_validation
+  configuration_source    = "terraform-alicloud-modules/terraform-alicloud-market-wordpress"
+}
+
+data "alicloud_regions" "this" {
+  current = true
+}
+
+module "ecs-instance" {
```
Comment (Rich-yi) [inline main.tf]: 我觉得这个主main里面用resource创建ecs好点

**Review by xiaozhu36 — APPROVED** [review]

**Pull request merged**